app/src/main/res/values/strings.xml

```diff
@@ -173,7 +173,11 @@ To display the map, vector tiles are retrieved from the &lt;a href=\"https://map
     <string name="autosync_off">"Off"</string>
     <string name="quest_streetSurface_title">"What surface does this piece of road have?"</string>
     <string name="quest_streetSurface_name_title">"What surface does the road \"%s\" have here?"</string>
+    <string name="quest_sideWalk_name_title">"What surface does the sidewalk \"%s\" have here?"</string>
```

Comment:
This made sense for streets, which often use the `name` tag (e.g. NE 15th St), but sidewalks should almost never have a `name` tag. If the quest module only shows this when a `name` tag is present, this shouldn't hurt, though.